test/spix/smoke_test.py

```diff
@@ -28,23 +28,35 @@ def test_start_app(app, screenshot_path, extra, process_alive):
     """
     assert app.existsAndVisible("mainWindow")
 
-    assert app.existsAndVisible("mainWindow/welcomeScreen")
+    app.setStringProperty("mainWindow", "width", "500")
+    app.setStringProperty("mainWindow", "height", "500")
+
+    time.sleep(2)
```

Comment:
What's that sleep for?

Reply:
I've seen intermittent failures otherwise.

Reply:
Don't remember the reason?

Reply:
It actually had to do with giving QField some time to load the project. I guess I can get rid of this one for the startup test.

Reply:
(actually no, we'd want to get a screen capture from the loaded project too, so let's keep the sleep time)

Reply:
We'll want to connect to a signal when rendering has finished?

Reply:
Eventually, whenever spix supports that :)

Reply:
It does not :fearful: ?
Then we can just do a param `finishedRendering` and check if this is set in a loop for now?

Reply:
(Discussed on chat)